util/golden-ratio/swm-golden-ratio.lisp

```diff
@@ -18,10 +18,10 @@
                                            (stumpwm:current-head)))))
 
 (defun resize-to-golden-ratio (to-frame from-frame)
-  (if *golden-ratio-on*
-      (let* ((target-x (target-px (stumpwm::head-width (stumpwm:current-head))))
-             (target-y (target-px (stumpwm::head-height (stumpwm:current-head)))))
-        (if *golden-ratio-on*
+  (if (not (stumpwm::single-frame-p))
```

Comment:
:thought_balloon: instead of nested IF forms consider using AND. `(when (and *golden-ratio-on* (not (stumpwm::single-frame-p)) ...)`

Because it is a single-armed conditional it would more appropriate to use WHEN instead of IF. While we are updating the code could remove the PROGN as the body of LET* is an implicit PROGN :pray:

Reply:
I didn't knew the part about `let*`! Thats pretty usefull.
Also, I didn't realize I had two `if` before you pointed it out